project_testing/testing_baseline.yml

```diff
@@ -25,6 +25,7 @@ workflow_generator:
       simulation_control_run_period_end_month: 12
       simulation_control_run_period_end_day_of_month: 31
       simulation_control_run_period_calendar_year: 2007
+      debug: true
```

Comment:
Why?

Reply:
I switched this on so that I could look at rdd files for good output variable candidates.

Reply:
It's also consistent with telling `server_directory_cleanup` to retain all files.